qutebrowser/browser/commands.py

```diff
@@ -985,6 +987,37 @@ def quickmark_load(self, name, tab=False, bg=False, window=False):
         url = objreg.get('quickmark-manager').get(name)
         self._open(url, tab, bg, window)
 
+    @cmdutils.register(instance='command-dispatcher', name='select-follow',
+                       scope='window')
```

Comment:
You should probably add `, hide=True` here as this isn't a command which should show up in the completion. Also the `name=` is unnecessary, as the function name `select_follow` gets converted to the command `select-follow` automatically.